types/forge-apis/src/viewer.d.ts

```diff
@@ -0,0 +1,485 @@
+//
+// Copyright (c) Autodesk, Inc. All rights reserved
+//
+// Permission to use, copy, modify, and distribute this software in
+// object code form for any purpose and without fee is hereby granted,
+// provided that the above copyright notice appears in all copies and
+// that both that copyright notice and the limited warranty and
+// restricted rights notice below appear in all supporting
+// documentation.
+//
+// AUTODESK PROVIDES THIS PROGRAM "AS IS" AND WITH ALL FAULTS.
+// AUTODESK SPECIFICALLY DISCLAIMS ANY IMPLIED WARRANTY OF
+// MERCHANTABILITY OR FITNESS FOR A PARTICULAR USE.  AUTODESK, INC.
+// DOES NOT WARRANT THAT THE OPERATION OF THE PROGRAM WILL BE
+// UNINTERRUPTED OR ERROR FREE.
+//
+/// <reference types="THREE" />
+
+declare namespace Autodesk {
```

Comment:
This is still just a namespace in the global scope; who is using these anyway, and how are they supposed to be consumed?

I just don't see where they exist in the real package.

Reply:
This is for our Model Visualization library [Viewer.js](
https://forge.autodesk.com/en/docs/viewer/v6/reference/javascript/) which at the moment can only be loaded via script tags so it’s not a part of our SDK.

But we included the typings for it because 1) it’s really badly wanted (see [here](
https://stackoverflow.com/questions/54220263/set-color-of-object-autodesk-forge/) for an example) by our community and 2) Viewer.js is arguably an indispensably component for the front end of most Forge projects.

Reply:
@DanielRosenwasser is there anything else I can help so we can go ahead to merge? Thanks!

Reply:
I see - would it make sense to have a separate package for Viewer.js that users can consume independently of `force-apis`?

Reply:
@DanielRosenwasser No it would not, for the reasons stated in my previous comments. If the package is not alright with you as it is now pls close this pull request and we will merge it with our real package coz while we took time to exchange comments for the past week or so hundreds of developers already missed out on these definitions that would have made their life a lot easier. Tks for your time.